nodejs/week1/src/backend/app.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+const reviews = require("./data/reviews.json");
+const warmup = require("./data/warmup.js");
+
+//-----the result of the of the circle measurements in the server----------------
+
+console.log(warmup); 
+
+//----------Meals sharing Web ---------------------------
+
+app.get("/", async (request, response) => {
+    response.send("Meals Sharing Web App");
+}); 
+
+
+//----------------Respond with the json for all the meals.---------------------
+
+app.get("/meals", async (request, response) => {
+    
+    response.send(module.exports = {meals,reviews});
```

Comment:
are you passing all the tests? i think you are missing the part, where you are adding the relevant reviews to the meal objects... read line 172 in homework.md

Reply:
Is true i forgot this point in the task. I will do